src/handlers/oai/verbs.js

```diff
@@ -91,7 +91,7 @@ const getRecord = async (url, id) => {
           },
           _text: url,
         },
-        GetRecord: { ...record },
+        GetRecord: { record: record },
```

Comment:
This can be shortened to 
```suggestion
        GetRecord: { record },
```

`{ ...record }` is an object with a copy of all of `record`'s fields in it. `{ record }` and `{ record: record }` are semantically equivalent. I'm happy leaving it as is, but given the specific diff behind this change I thought it was worth mentioning.